pkgs/development/tools/wails/default.nix

```diff
@@ -58,18 +59,18 @@ buildGoModule rec {
   # As Wails calls a compiler, certain apps and libraries need to be made available.
   postFixup = ''
     wrapProgram $out/bin/wails \
-      --prefix PATH : ${lib.makeBinPath [ pkg-config go gcc nodejs ]} \
-      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ gtk3 webkitgtk ]} \
+      --prefix PATH : ${lib.makeBinPath [ pkg-config go stdenv.cc nodejs ]} \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath ([ ] ++ lib.optionals stdenv.isLinux [ gtk3 webkitgtk ])}" \
```

Comment:
don't need  `[ ]++`  lib.optionals returns `[ ]` when first argument is false

Reply:
Thanks, @kirillrdy, that's good to know 👍 I've update this PR accordingly.